### PR TITLE
Add remaining resources to returns swagger

### PR DESCRIPTION
**Pull request by bc-luke**
# [SHIPPING-1335](https://jira.bigcommerce.com/browse/SHIPPING-1335)

## What changed?

* Add remaining resources to returns swagger

### CODE REVIEW

File: beta-documentation/returns-api-PROJECT-2776/returns.json [hunk 1]
```diff
@@ -90,6 +349,361 @@
...
+          "format": "decimal",
+          "description": "The value of the item"
+        },
+        "preferred_outcome": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "integer",
+              "format": "int64",
+              "description": "Unique identifier of the preferred outcome",
```
Comment (TomA-R) [inline beta-documentation/returns-api-PROJECT-2776/returns.json]:
```suggestion
              "description": "Unique identifier of the preferred outcome"
```

Comment (bc-luke) [reply beta-documentation/returns-api-PROJECT-2776/returns.json]: Good eyes!

**Comment by tatiana-perry** [comment]
@bc-luke @TomA-R I merged this, I think either of you should be able to both approve and merge the changes. If not please let me know so I can get the permissions changed.

**Comment by bc-luke** [comment]
Thanks @tatiana-perry

**Pull request merged**